src/FontLib/Glyph/OutlineSimple.php

```diff
@@ -265,7 +265,7 @@ public function getSVGContours($points = null) {
       $points = $this->points;
     }
 
-    $length     = count($points);
+    $length     = (empty($points) ? 0 : count($points));
```

Comment:
I think we should `return $path` if `$points` is empty, it would avoid the loop : even if it's not looping, we should escape sooner.